Content.Server/Radio/Components/HeadsetComponent.cs

```diff
@@ -1,19 +1,43 @@
 using Content.Server.Radio.EntitySystems;
 using Content.Shared.Inventory;
-using Content.Shared.Radio;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
+using Robust.Shared.Audio;
+using Robust.Shared.Containers;
 
 namespace Content.Server.Radio.Components;
-
 /// <summary>
 ///     This component relays radio messages to the parent entity's chat when equipped.
 /// </summary>
 [RegisterComponent]
 [Access(typeof(HeadsetSystem))]
 public sealed class HeadsetComponent : Component
 {
-    [DataField("channels", customTypeSerializer: typeof(PrototypeIdHashSetSerializer<RadioChannelPrototype>))]
-    public readonly HashSet<string> Channels = new() { "Common" };
+    //[DataField("channels", customTypeSerializer: typeof(PrototypeIdHashSetSerializer<RadioChannelPrototype>))]
```

Comment:
Need serializer.

Reply:
Why? it is not datafield anymore...

Reply:
Does it? It is filling only by encryption chips that is already serialized by this prototype